modules/nf-commons/build.gradle

```diff
@@ -38,3 +38,4 @@ dependencies {
     testImplementation(testFixtures(project(":nextflow")))
 }
 
+test.dependsOn(":nf-plugin-template:makeZip")
```

Comment:
Is this really needed here?

Reply:
this is the way I found to have updated the plugin before running tests automatically. With this hack before to run a test in nf-commons every change in nf-plugin-template is tested